starport_template/lib/model/tx_json.dart

```diff
@@ -0,0 +1,59 @@
+class Tx {
+  Tx({
+    required this.type,
+    required this.body,
+  });
+
+  String type;
+  Body body;
+
+  factory Tx.fromJson(Map<String, dynamic> json) => Tx(
+        type: json["@type"] as String? ?? '',
+        body: Body.fromJson(json["body"] as Map<String, dynamic>),
+      );
+}
+
+class Body {
```

Comment:
```suggestion
class TxBodyJson {
```
also, should be in separate file